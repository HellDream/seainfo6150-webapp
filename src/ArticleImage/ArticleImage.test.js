import React from "react";
import ArticleImage from "./ArticleImage";

describe("ArticleImage tests", () => {
  it("renders correctly", () => {
    const { container } = render(<ArticleImage url="http://www.123.com" title="hello" />);
    expect(container).toMatchSnapshot();
  });
});
