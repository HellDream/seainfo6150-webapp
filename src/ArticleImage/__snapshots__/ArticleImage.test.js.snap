// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage tests renders correctly 1`] = `
<div>
  <img
    alt="hello"
    class="image"
    src="http://www.123.com"
  />
</div>
`;
